### DISCUSSION TITLE
Where is data saved? sqlite databse? wheres i the file?

### DISCUSSION THREAD

**Discussion by leoborlot** [usememos/memos#504] [General]
hi guys. 

wheres is the data saved? How may I backup the information?

**Comment by boojack** [comment]
If you start memos with the suggested command:

```docker
docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest
```

then the SQLite database file will be `~/.memos/memo_prod.db`. So you can backup this file as you want.

**Reply by VlaK0r** [reply]
Help please. I can't find this file.
